public/src/components/channelManagement/campaigns/ChannelCard.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { makeStyles, Theme } from '@material-ui/core';
+import TestCard from './TestCard';
+import { TestChannelItem } from './CampaignsEditor';
+import { CombinedTest } from './CampaignsForm';
+
+const useStyles = makeStyles(({ spacing }: Theme) => ({
+  channelContainer: {
+    marginBottom: spacing(4),
+    paddingTop: spacing(2),
+  },
+  noTestsWarning: {
+    marginLeft: spacing(4),
+  },
+  channelTitle: {
+    marginBottom: spacing(2),
+    fontSize: '18px',
+    fontWeight: 500,
+  },
+}));
+
+interface ChannelCardProps {
+  channelData: TestChannelItem;
+  tests: CombinedTest[];
+}
+
+function ChannelCard({ channelData, tests }: ChannelCardProps): React.ReactElement {
+  const classes = useStyles();
+
+  const getKey = (test: CombinedTest) => {
+    return `${channelData.name}|${test.name}`;
+  };
+
+  return (
+    <div className={classes.channelContainer}>
+      <div className={classes.channelTitle}>{channelData.name} channel</div>
+      {tests.length > 0 ? (
+        tests.map(test => (
+          <TestCard
+            test={test}
+            keyId={`${getKey(test)}_LINK`}
+            linkPath={channelData.link}
+            key={`${getKey(test)}_CARD`}
+          />
+        ))
+      ) : (
+        <div className={classes.noTestsWarning}>No tests have been set up for this channel</div>
```

Comment:
Pretty minor thing but the warning messages are not quite consistent in terms of punctuation and case:
`No tests have been set up for this channel`
`No variants have been created for this test!`
`No locations have been selected for this Test`

Reply:
Fixed:
`No Tests have been set up for this Channel.`
`No Variants have been created for this Test.`
`No locations have been selected for this Test.`